device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClient.java

```diff
@@ -379,7 +379,7 @@ private DeviceClient(String uri, String deviceId, SecurityProvider securityProvi
         commonConstructorSetup();
     }
 
-    void closeFileUpload() throws IOException
```

Comment:
Is this a breaking change?

Reply:
No, this method isn't accessible to users